tests/pgautofailover_utils.py

```diff
@@ -1545,15 +1546,35 @@ def list_replication_slot_names(self):
         """
         Returns a list of the replication slot names on the local Postgres.
         """
+        slots = {}
+        nameList = []
+
         query = (
-            "select slot_name from pg_replication_slots "
+            "select oid, slot_name from pg_replication_slots "
```

Comment:
Surprisingly, I could complete some of the unit testing on my local setup, and now I realise that the `pg_replication_slots` view does not have an OID column, and replication slots are identified with their name, I don't see that they would expose an OID. Cancelling that PR.